src/dispatch/messaging.py

```diff
@@ -231,6 +231,27 @@ class MessageType(str, Enum):
 INCIDENT_TASK_RESOLVED_DESCRIPTION = """
 The following incident task has been resolved in the incident document.\n\n*Description:* {{task_description}}\n\n*Assignees:* {{task_assignees|join(',')}}"""
 
+INCIDENT_WORKFLOW_CREATED_DESCRIPTION = """
+A new workflow instance has been created.
+\n\n *Creator:* {{instance_creator_name}}
+"""
+
+INCIDENT_WORKFLOW_UPDATE_DESCRIPTION = """
+This workflow's status has changed from *{{ instance_status_old }}* to *{{ instance_status_new }}*.
+\n\n*Workflow Description*: {{workflow_description}}
+\n\n *Creator:* {{instance_creator_name}}
+"""
+
+INCIDENT_WORKFLOW_COMPLETE_DESCRIPTION = """
```

Comment:
Should this one extend `INCIDENT_WORKFLOW_UPDATE_DESCRIPTION` since it's the same except for the artifacts part? Or perhaps rename and only use `INCIDENT_WORKFLOW_COMPLETE_DESCRIPTION` since it seems it can handle both cases?

Reply:
I think the data they will display will diverge relatively soon but for this first pass they are very similar.